app/views/shared/metadata/edit/_study.html.erb

```diff
@@ -33,6 +33,7 @@
     <% metadata_fields.with_options(grouping: 'Data Release') do |group| %>
       <%= metadata_fields.text_field(:data_access_group) %>
       <%= group.select_by_association(:data_release_study_type)%>
+      <%= help_link 'Introduction to Accessioning', 'Accessioing', class: 'offset-md-2' %>
       <% strategies = Study::DATA_RELEASE_STRATEGIES.map { |s| [t(s.downcase.gsub(' ','_'),scope:'metadata.study.metadata.data_release_strategy.values'),s ] } %>
```

Comment:
The spelling error here is in the name of the help page. I'm not sure its possible to change it without breaking any existing links.